client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/InteractedContactAccordion.tsx

```diff
@@ -35,10 +36,27 @@ const InteractedContactAccordion = (props: Props) => {
         shouldDisable,
     } = props;
 
+    const getAccordionClasses = () : string => {
+        let classesList : string[] = [];
+        classesList.push(classes.accordion);
+
+        if(errors.form) {
+            const currentErrors = errors.form[index];
+            if(currentErrors !== undefined){
+                for(let key in currentErrors){
+                    if(currentErrors[key] !== undefined) {
+                        classesList.push(classes.errorAccordion)
```

Comment:
Why push the same class multiple times? Maybe .some() would be helpful here instead? 🤔

Reply:
but .some is for arrays, this is an object , so I'm iterating over it's keys

Reply:
could do .some on Object.keys or Object.entries

Reply:
tnx , fixed [here](7f40897)

Reply:
fe9e82b does this works?

Reply:
thats a schema update on a different file 🤷‍♀️

Reply:
urghhhh , [its here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/998/commits/b5eadd1aae481db271e6fe322272911dae1d5a29) <sub>i hope</sub>

Reply:
yup!